bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractComponentImpl.java

```diff
@@ -143,7 +142,8 @@ private void init() {
     public String getId() {
         if (id == null) {
             String resourceCallerPath = request != null ? (String) request.getAttribute(REQ_ATTR_RESOURCE_CALLER_PATH) : null;
-            this.id = ComponentUtils.getId(this.resource, this.currentPage, resourceCallerPath, this.componentContext);
+            this.id = com.adobe.cq.wcm.core.components.util.ComponentUtils.getId(this.resource, this.currentPage, resourceCallerPath,
```

Comment:
Please remove the changes which are not required, please mark the PR for review only if it is ready